src/collections/components/CollectionList/CollectionList.tsx

```diff
@@ -68,107 +69,130 @@ const CollectionList = withStyles(styles, { name: "CollectionList" })(
+              />
+            </TableCell>
+            <TableCell className={classes.colAvailability}>
+              <FormattedMessage
+                defaultMessage="Availability"
+                description="collection availability"
+                id="collectionListAvailabilityColumnHeader"
+              />
+            </TableCell>
+          </TableHead>
+          <TableFooter>
+            <TableRow>
+              <TablePagination
+                colSpan={numberOfColumns}
+                settings={settings}
+                hasNextPage={
+                  pageInfo && !disabled ? pageInfo.hasNextPage : false
+                }
+                onNextPage={onNextPage}
+                onUpdateListSettings={onUpdateListSettings}
+                hasPreviousPage={
+                  pageInfo && !disabled ? pageInfo.hasPreviousPage : false
+                }
+                onPreviousPage={onPreviousPage}
+              />
+            </TableRow>
+          </TableFooter>
+          <TableBody>
+            {renderCollection(
+              collections,
+              collection => {
+                const isSelected = collection
+                  ? isChecked(collection.id)
+                  : false;
+                return (
+                  <TableRow
+                    className={classes.tableRow}
+                    hover={!!collection}
+                    onClick={collection ? onRowClick(collection.id) : undefined}
+                    key={collection ? collection.id : "skeleton"}
+                    selected={isSelected}
+                  >
+                    <TableCell padding="checkbox">
+                      <Checkbox
+                        checked={isSelected}
+                        disabled={disabled}
+                        disableClickPropagation
+                        onChange={() => toggle(collection.id)}
+                      />
+                    </TableCell>
+                    <TableCell className={classes.colName}>
+                      {maybe<React.ReactNode>(
+                        () => collection.name,
+                        <Skeleton />
+                      )}
+                    </TableCell>
+                    <TableCell className={classes.colProducts}>
+                      {maybe<React.ReactNode>(
+                        () => collection.products.totalCount,
+                        <Skeleton />
+                      )}
+                    </TableCell>
+                    <TableCell className={classes.colAvailability}>
+                      {maybe(
+                        () => (
+                          <StatusLabel
+                            status={
+                              collection.isPublished ? "success" : "error"
+                            }
+                            label={
+                              collection.isPublished
+                                ? intl.formatMessage(commonMessages.published)
```

Comment:
We cannot share those as it will need to be properly gendered ("opublikowany" vs. "opublikowana") depending on the context (product/category/collection can all have different genders in different languages).